comps/dataprep/redis/langchain/prepare_doc_redis.py

```diff
@@ -170,11 +170,19 @@ def ingest_data_to_redis(doc_path: DocPath):
 
     content = document_loader(path)
 
-    chunks = text_splitter.split_text(content)
+    structured_types = [".xlsx", ".csv", ".json", "jsonl"]
```

Comment:
There are two more redis related code also need to change in this way.

Reply:
code for ray is fixed. code for llamaindex uses another data loader will not be effected